src/content/fr/2019/media.md

```diff
@@ -0,0 +1,416 @@
+---
+part_number: I
+chapter_number: 4
+title: Media
+description: Chapitre Média du Web Almanac 2019 couvrant les tailles et formats des fichiers d’images, les images adaptatives, les Indications Client, le lazy loading, l’accessibilité et la vidéo.
```

Comment:
Curious why you choose to translate "Client Hints" but not "lazy loading"?

Happy to merge as is but just curious for my own understanding.

Reply:
I feel that the term "Lazy Loading" is already present in French Front-End conversations, as the technique has been around for years. There's no need to use the French language to help people understand what it is (in France's French, that is, maybe @arigaud-ca would disagree).
The term "Client Hints" is more recent, has no momentum in French (except in some niches). Translating it (while keeping the English term next to it) can help people who are discovering the subject to understand it faster.

Reply:
OK. Thanks for explaining!